Yoast/wordpress-seo
# FRO-124 Feature/admin redesign

## Context


* Implements the redesign of the admin pages

## Summary


This PR can be summarized in the following changelog entry:

* Implements the redesign of the admin pages

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and the latest develop from the monorepo
* Check if the redesign is correctly implemented
* Check if all functionality still works as it should

## UI changes

## Documentation

## Quality assurance


Fixes #
